frontend/package.json

```diff
@@ -118,49 +111,33 @@
         ]
     },
     "devDependencies": {
-        "@cypress/code-coverage": "^3.13.9",
         "@istanbuljs/nyc-config-typescript": "^1.0.2",
         "@jackstenglein/eslint-config-dojo": "file:../eslint-config-dojo",
         "@testing-library/jest-dom": "^6.6.3",
         "@testing-library/react": "^16.1.0",
-        "@testing-library/user-event": "^14.5.2",
-        "@types/command-line-args": "^5.2.3",
-        "@types/command-line-usage": "^5.0.4",
         "@types/lodash.debounce": "^4.0.9",
         "@types/luxon": "^3.4.2",
-        "@types/proper-lockfile": "^4.1.4",
         "@types/react-copy-to-clipboard": "^5.0.7",
-        "@types/react-grid-layout": "^1.3.5",
         "@types/react-resizable": "^3.0.8",
         "@types/uuid": "^10.0.0",
         "@typescript-eslint/eslint-plugin": "^7.12.0",
         "@typescript-eslint/parser": "^7.12.0",
-        "babel-plugin-istanbul": "^7.0.0",
-        "command-line-args": "^6.0.1",
-        "command-line-usage": "^7.0.3",
         "cypress": "^13.17.0",
         "cypress-parallel": "^0.14.0",
-        "dependency-cruiser": "^16.5.0",
```

Comment:
Didn't you just add dependency-cruiser a few PRs ago? Do we no longer need it?

Reply:
Oh my goodness, good catch.

Reply:
I did, but you launch it with npx according to the docs, so it's not needed. The important part is the config that was generated. We haven't integrated it fully into our deployment process yet.